ooiui/core/routes/common.py

```diff
@@ -252,9 +256,8 @@ def get_operator_event_type():
     response = requests.get(app.config['SERVICES_URL'] + '/operator_event_type', params=request.args)
     return response.text, response.status_code
 
-
 @app.route('/api/uframe/glider_tracks', methods=['GET'])
 def get_glider_track():  
     token = get_login()
     response = requests.get(app.config['SERVICES_URL'] + '/uframe/get_glider_track/'+request.args['id'], auth=(token, ''), data=request.args)
-    return response.text, response.status_code
```

Comment:
That was only removed because he hadnt updated to your most recent changes... he needs to update that (rebase?)

@brian-smith723 You need to include only the files and lines of code you want to commit.. you've included a file/line that you dont want in here.